foxy/distribution.yaml

```diff
@@ -1747,8 +1747,8 @@ repositories:
       - gpsd_client
       tags:
         release: release/foxy/{package}/{version}
-      url: https://github.com/swri-robotics-gbp/gps_umd-release.git
-      version: 1.0.6-1
+      url: https://github.com/ros2-gbp/gps_umd-release.git
+      version: 1.0.8-1
```

Comment:
It doesn't look like the new release repository contains this tag 1.0.8-1, it seems like the latest is 1.0.4 https://github.com/ros2-gbp/gps_umd-release#gps_umd-foxy---104-1, it's not as up-to-date as https://github.com/swri-robotics-gbp/gps_umd-release.git which at least had 1.0.6.

But maybe I'm missing something